### PR TITLE
Login for fandom shop

### PR DESCRIPTION
**Pull request by vikiival**

### CODE REVIEW

**Comment by vikiival** [comment]
Btw @Hladikes feel free to commit here

**Comment by Hladikes** [comment]
There is currently this issue, which only occurs on product subpage, and only when deployed 🫠
I would guess that the bundle is not suited for browser but, that's not true since it does work on the main page

image

File: import_map.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "@kodadot1/uniquery": "https://esm.sh/@kodadot1/uniquery@0.3.1-rc.0",
     "@kodadot1/uniquery/": "https://esm.sh/@kodadot1/uniquery@0.3.1-rc.0/",
     "@kodadot1/static": "https://esm.sh/@kodadot1/static@0.0.3-rc.4",
-    "@kodadot1/static/": "https://esm.sh/@kodadot1/static@0.0.3-rc.4/"
+    "@kodadot1/static/": "https://esm.sh/@kodadot1/static@0.0.3-rc.4/",
+    "extension-dapp": "./utils/extension-dapp.js"
```
Comment (vikiival) [inline import_map.json:19]: maybe because the route is not correct?
or because the import map?

Comment (Hladikes) [reply import_map.json:19]: I had the same error even before putting the lib in the import_map. And route itself is working, everything is working just fine in dev enviroment

Comment (Hladikes) [reply import_map.json:19]: Maybe I can try to use esbuild again, but I will try different module type (maybe UMD) and I will try to load it with `<script src="">` globally 🤷‍♂️

Comment (vikiival) [reply import_map.json:19]: maybe, i would try to update fresh 🤷

Comment (Hladikes) [reply import_map.json]: didn't work :(

Comment (Hladikes) [reply import_map.json]: update: the issue is not related to this branch and code changes related to this branch. Open https://fandom.kodadot.art/products/30-2 and check the dev tools

**Comment by vikiival** [comment]
@Hladikes can you please resolve conflicts?

**Pull request merged**